clients/bazel-module-registry/src/main/kotlin/Model.kt

```diff
@@ -24,16 +24,16 @@ import io.ks3.java.typealiases.UriAsString
 import kotlinx.serialization.Serializable
 
 /**
- * E.g. https://bcr.bazel.build/modules/glog/metadata.json.
+ * Schema: https://github.com/bazelbuild/bazel-central-registry/blob/main/metadata.schema.json.
```

Comment:
Please use a permalink to document the exact state of the schema you were using when writing this.

Another nit: We usuall just write "See https://..." in similar cases.

Reply:
How to deal with a too long link then?

Reply:
I believe HTTP links may exceed the line length limit, see e.g.

https://github.com/oss-review-toolkit/ort/blob/5228030136c548d67ae0818922dfd066793d72b4/plugins/package-managers/bundler/src/main/kotlin/Model.kt#L36

Reply:
Also you may shorten the SHA1 in such links down to 7 chars.